js/hacks/hack2_PRINTER_SOURCE_CODE/hack2_story.js

```diff
@@ -193,21 +190,18 @@ monogatari.script ({
 		"e Clara, we are gathering people to start protests and even striking.",

```

Comment:
For continuations sake in regards to Yu Xi's spacing comments, there's weird spacing here as well

Reply:
I think if you look at the changes from the perspective of my latest (as of 2pm) commit (7e3ec037) you'll see that this one is aligned.